src/test/java/com/epam/tc/hw1/DivTest.java

```diff
@@ -0,0 +1,29 @@
+package com.epam.tc.hw1;
+
+import static org.assertj.core.api.AssertionsForClassTypes.assertThat;
+
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Test;
+
+public class DivTest  extends AbstractTest {
+
+    @Test
+    public void divTest() {
+        double res = calculator.div(4, 2);
+        assertThat(res).isEqualTo(2);
+    }
```

Comment:
Use dataproviders

_6.	All test data should be provided to test through the DataProviders_

Reply:
I removed simply tests and expanded the data set in the existing ones through the DataProviders